pages/windows/choco-new.md

```diff
@@ -1,24 +1,24 @@
 # choco new
 
 > Generate new package specification files with Chocolatey.
-> More information: <https://chocolatey.org/docs/commands-new>.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/new>.
 
-- Create a new package skeleton:
+- Create the new package skeleton:
```

Comment:
I'm sure that here should be "a" rather than "the" because it doesn't matter which package skeleton one creates.

Reply:
```suggestion
- Create a new package skeleton:
```